package.json

```diff
@@ -34,6 +35,7 @@
     "eslint-plugin-react": "^7.20.0",
     "eslint-plugin-wpcalypso": "^4.1.0",
     "fs-extra": "^9.0.1",
+    "jest": "26.1.0",
```

Comment:
We probable don't need to touch dependencies. This package appears to rely on `@wordpress/scripts`, we might as well use that tooling for testing:

https://github.com/WordPress/gutenberg/blob/master/packages/scripts/README.md

Reply:
:+1: